accelerator/cpu_accelerator.py

```diff
@@ -234,6 +234,9 @@ def on_accelerator(self, tensor):
 
     # create an instance of op builder and return, name specified by class_name
     def create_op_builder(self, op_name):
+        # avoid abstract classes
```

Comment:
@hipudding, can you please explain these changes? 

@delock, FYI

Reply:
Sure, of course.

I moved no_impl class out of cpu folder as a public class for every backend. If any backend didn't have any ops, ds_report will not print any op (including NotImplementedBuilder, because the op_builder/{new backend} folder is empty, ds_report can't find any modules). For cpu accelerator, if any ops is not implement, it will fallback to NotImplementedBuilder, but it didn't skip the abstract classes(not regular ops), which means if ds_report scanning module named OpBuilder or CPUOpBuilder, it will still return the fallback op(NotImplementedBuilder), and it will print NotImplementedBuilder in ds_report result.

1. Considering cuda_accelerator skip abstract class, I make them behave the same. See cuda_acclerator._lazy_init_class_dict
2. NotImplementedBuilder is not considered as a normal op any more, it should not show in ds_report.
3. Make every backend behave the same no matter it has any op_builders or not.

Reply:
@tjruwase @hipudding  I'm more favor to decouple accelerator backends than reuse between accelerator backends.   Since it's hard for a developer to test on all the backends, decouple would avoid cross interfere between accelerator changes.  I I think its better each accelerator backend implement its own NotImplementedBuilder, even the logic is almost the same.

Reply:
I agree woth @delock, I will revert the last commit and leave a NotImplementBuilder for NPU.

Reply:
Fixed according to @delock 's suggestion.